source/user-guide/how-to-update-pagerduty-status-page.html.md.erb

```diff
@@ -0,0 +1,32 @@
+---
+owner_slack: "#modernisation-platform"
+title: How to update the pagerduty status page in the Modernisation Platform
+last_reviewed_on: 2023-11-02
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+## Introduction
+Sometime you may need to update the external status page for the modernisation platform [modernisation platform status page](https://status.modernisation-platform.service.justice.gov.uk/) this guide will show you how to do this.
+
+### Manually creating and updating status changes
+Status changes can be created in two ways the first way is by logging on to the pagerduty site and the other is by using the pugerduty incident creation app within slack via the /PD command.
```

Comment:
Can you create a status without an incident?

Reply:
talked about this in standup